templates/typescript_gapic/namer.js

```diff
@@ -20,6 +20,19 @@
  * the template engine.
  */
 
+const STRICT_MODE_RESERVED_WORDS = new Set([
```

Comment:
Did we recently turn on strict mode, or did we just happen to have not bumped into this issue before?

Reply:
As I know, we always use `strict mode`. I think it just bumped this issue this time.